planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -75,10 +74,22 @@
     public static final String SEACLOUDS_APPLICATION_INFORMATION_POLICY_TYPE = "seaclouds.policies.app.information";
     public static final String SEACLOUDS_APPLICATION_POLICY_NAME = "seaclouds.app.information";
     public static final String SEACLOUDS_NODE_PREFIX = "seaclouds.nodes";
+    public static final String SEACLOUDS_NODE_TYPES = "seaclouds-types";
+    public static final String SEACLOUDS_NODE_TYPES_VERSION = "0.8.0-SNAPSHOT";
 
     private static DeployerTypesResolver deployerTypesResolver;
     static Map<String, MonitoringInfo> monitoringInfoByApplication=new HashMap<>();
 
+    /**
+     *
+     * @param adp YAML string containing the ADP
+     * @param monitorGenURL URL of the Monitoring service
+     * @param monitorGenPort Port of the Monitoring service
+     * @param slaGenURL URL of the SLA service
+     * @param influxdbURL URL of InfluxDB
+     * @param influxdbPort Port of InfluxDB
+     * @return YAML string containing the generated DAM
```

Comment:
I am sure it is possible, because DAM will be processed by brooklyn-tosca which will use `ArchiveRoot`.